test/Interop/Cxx/stdlib/use-std-optional.swift

```diff
@@ -16,7 +16,7 @@ StdOptionalTestSuite.test("pointee") {
   let pointee = nonNilOpt.pointee
   expectEqual(123, pointee)
 
-#if !os(Linux) // crashes on Ubuntu 18.04 (rdar://113414160)
+#if !os(Linux) && !os(FreeBSD) // crashes on Ubuntu 18.04 (rdar://113414160)
```

Comment:
If this is crashing on FreeBSD as well, can you file a GitHub issue or Radar as appropriate and reference it here too?